webpack/scenes/ContentViews/Details/ContentViewDetailActions.js

```diff
@@ -51,6 +57,37 @@ export const updateContentView = (cvId, params) => async dispatch => dispatch(pu
   },
 }));
 
+export const addComponent = params => async dispatch => dispatch(put({
```

Comment:
`async` is not needed

Reply:
Why is this a thunk? So you can have access to `dispatch` inside the success and error handlers?  There must be another way..

Reply:
Refactored the code to remove the need for the thunk..Added the table refresh as an additionalListener to the Table wrapper..